src/test/objectstore/test_bluefs.cc

```diff
@@ -679,6 +705,32 @@ TEST(BlueFS, test_compaction_async) {
     writes_done = true;
     join_all(sync_threads);
     fs.compact_log();
+
+    {
+      string dir = "dir.after_compact_test";
+      ASSERT_EQ(0, fs.mkdir(dir));
```

Comment:
We are using the same Dir/File names like the test_compaction_sync().
If that is intentional then they should all be defined in a global scope.
It seems that you trust that the async test will always be called after the sync test as test_compaction_async() opens File in a Dir which was created in test_compaction_sync().
What will happen if we will only run test_compaction_async()?
Also, what will happen if we would run test_compaction_sync() twice? Will we fail because Dir already exists?
I think it would be better to use different Dir/File and let each function create/delete its Dir/File

Reply:
Each test is creating own separate bluefs instance.
`ASSERT_EQ(0, fs.mkfs(fsid, { BlueFS::BDEV_DB, false, false }));`

Reply:
I am personally against having globals in unittest.
Even if we create test by copy-paste and end up with same values, I am ok with it, as each unit test should be able to run in isolation.